.vscode/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "cSpell.words": [
+        "holoport",
+        "hpos"
+    ]
+}
```

Comment:
let's add `.vscode/` to `.gitignore`, or actually not. Maybe you can [add it to your .global_gitignore](https://anthonygharvey.com/how-to-create-a-global-gitignore-file)?